.github/workflows/validate.yml

```
@@ -3,20 +3,33 @@ name: Scheduled validator
 on: [push, pull_request]
 
 jobs:
-  validate:
-    name: OPTIMADE validator
+
+  pytest:
+    name: Sanity checks (pytest)
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: 3.7
     - name: Install OPTIMADE
       run: |
         python -m pip install --upgrade pip
-        pip install optimade
-        pip install pytest
-    - name: Run validator on repository
-      run: |
-        py.test
+        pip install -U setuptools
+        pip install optimade pytest
+    - name: Run local tests on repository
+      run: pytest
+
+  validator:
+    name: OPTIMADE validator
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: Materials-Consortia/optimade-validator-action@v2
+      with:
+        index: yes
+        protocol: https
+        domain: deploy-preview-${{ github.event.number }}--optimade-providers.netlify.app
```

Can we try this as a temporary fix? (Of course, it should work until we release OPTIMADE version 2...)

```suggestion
        domain: deploy-preview-${{ github.event.number }}--optimade-providers.netlify.app/v1
```

---

Sure. just pushed it, didn't notice your suggestion here, sorry.